docs/dapps/verify-airnode-addresses.md

```diff
@@ -1,41 +0,0 @@
----
-title: Verify Airnode addresses
-pageHeader: dApps → Integration
-outline: deep
----
-
-<PageHeader/>
-
-# Verify Airnode addresses
```

Comment:
We're still linking to this page from the market in the source feed tooltip. This is also present in the [new designs](https://www.figma.com/design/gWtbAejMNAUuOt4soabpf2/%E2%9F%81---Api3-Market-2.1---Branding-Update?node-id=6-87372&p=f&t=opuvciigrQohQAKz-0).

<img width="654" alt="image" src="https://github.com/user-attachments/assets/3614f533-f23d-4b59-8d10-47baa240684b" />

I think there should be some docs section explaining how the verification works, because the user doesn't understand what the external link means (e.g. [this](https://signed-api.kaiko.com/))

Reply:
Right, it should link https://docs.api3.org/dapps/integration/#verifying-first-party-sources
I was meaning to rewrite that tooltip anyway so let me go and do that now